tests/config/nominal/.wakeroot

```diff
@@ -1,3 +1,5 @@
 {
-  "user_config": "./wake_user.json"
+  "user_config": "./wake_user.json",
+  "log_header":"",
+  "log_header_extra_width":0
```

Comment:
nominal test should set custom values and ensure they are pulled through

Reply:
I guess I'm not sure what you mean by that?

Reply:
I just want `log_header` and `log_header_extra_width` to have some value a random repo would set. Right now all of our tests for config only test the "empty" values or the default values